charts/renku/values.yaml

```diff
@@ -88,7 +88,7 @@ keycloak:
           - -c
           - |
             echo "Copying theme..."
-            cp -R /renku-theme/* /theme
+            cp -R /renku_theme/* /theme
```

Comment:
:boom:

Reply:
it seems consistent with https://github.com/SwissDataScienceCenter/keycloak-theme/blob/master/Dockerfile#L2